.travis.yml

```diff
@@ -15,13 +15,9 @@ matrix:
     - python: 3.6
       env: TOX_ENV=isort
     - python: 3.6
-      env: TOX_ENV=py36-django-111
-    - python: 3.6
-      env: TOX_ENV=py36-django-19
-    - python: 3.5
```

Comment:
can we keep python 3.5 support?
This will allow us to merge this now and use it in the current version of docs italia

Reply:
Done 👍

Reply:
:bowing_man: